### PR TITLE
[console][table] adjust width of colspanned cell.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #18627 |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

File: src/Symfony/Component/Console/Helper/Table.php [hunk 1]
```diff
@@ -521,6 +521,19 @@ private function getColumnWidth($column)
                 continue;
             }
 
+            // split the TableCell centent in order to extending width of other columns
```
Comment (javiereguiluz) [inline src/Symfony/Component/Console/Helper/Table.php]: I don't understand what does this comment try to explain. We extend this cell up to the width of the largest column?

Comment (aitboudad) [inline src/Symfony/Component/Console/Helper/Table.php]: Normally when we use `colspan`, the cells are merged into one and other cells are [left blank](https://github.com/aitboudad/symfony/blob/issues_18627/src/Symfony/Component/Console/Helper/Table.php#L441-L442), 
but when we want to calculate the column width we should split the merged cells content in different emptied cells in order to assume the average width of each columns

**Comment by fabpot** [comment]
Thank you @aitboudad.

**Comment by fabpot** [comment]
@aitboudad Can you submit a PR for Symfony 2.8? I've reverted the change as I was not able to easily resolve the conflicts, the code is very different. Thanks.

**Comment by aitboudad** [comment]
ok

**Comment by aitboudad** [comment]
@fabpot see https://github.com/symfony/symfony/pull/18774

**Pull request merged**